.github/workflows/build.yml

```diff
@@ -167,7 +167,7 @@ jobs:
         uses: actions/checkout@v2.4.0
 
       - name: Setup Flutter environment
-        uses: subosito/flutter-action@v1.5.3
```

Comment:
i think we should pin the version to avoid stuff breaking because of new versions.

Reply:
I don't disagree as such. However we also be fine to lock into just the major or minor version as you wouldn't normally expect breaking behaviour between patches .